.travis.yml

```diff
@@ -35,7 +35,7 @@ install:
   - export LD_LIBRARY_PATH=$HOME/miniconda/envs/test-environment/lib/:$LD_LIBRARY_PATH
 
   # install TensorFlow (CPU version).
```

Comment:
In TF2.1, GPU support is provided by default I reckon. 
If we can confirm that TF2.0 works, it may be worth checking the change log of the 2.1.